README.md

```diff
@@ -634,15 +634,15 @@ All-in-one project, editor, file switcher, and preview with a horizontal side-by
-| `resizable`          | `boolean`                        | `false`                   | Allow dragging the line between editor and preview to change relative sizes                                   |
-| `sandboxBaseUrl`     | `string`                         | _module parent directory_ | Base URL for untrusted JavaScript execution (⚠️ use with caution, see [sandbox security](#sandbox-security)). |
-| `pragmas`            | `"on" \| "off" \| "off-visible"` | `"on"`                    | How to handle `playground-hide` and `playground-fold` comments ([details](#hiding--folding)).                 |
+| Name                 |  Type                            | Default                   | Description                                                                                                                                                                                        |
+| -------------------- | -------------------------------- | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `projectSrc`         | `string`                         | `undefined`               | URL of the [project manifest](#project-manifest) to load                                                                                                                                           |
+| `config`             | `ProjectManifest`                | `undefined`               | Get or set the project configuration and files, ([details](#option-3-config-property)).                                                                                                            |
+| `lineNumbers`        | `boolean`                        | `false`                   | Render a gutter with line numbers in the editor                                                                                                                                                    |
+| `editableFileSystem` | `boolean`                        | `false`                   | Allow adding, removing, and renaming files                                                                                                                                                         |
+| `resizable`          | `boolean`                        | `false`                   | Allow dragging the line between editor and preview to change relative sizes                                                                                                                        |
+| `sandboxBaseUrl`     | `string`                         | _module parent directory_ | Base URL for untrusted JavaScript execution (⚠️ use with caution, see [sandbox security](#sandbox-security)). Resolved relative to the module containing the definition of `<playground-project>`. |
```

Comment:
Clarifying my understanding. Is the advantage of this over `location.href` that `meta.url` is more unique and specific to the executed code location. Does this lead to fewer url collisions?

Reply:
The `sandboxBaseUrl` property tells playground where to look for the service worker script. By default, it's set to an unpkg.com URL, because of security (separate origin). However, it's useful during development to instead point at the local version, so that we can test changes to the service worker. This local version can be found in the root of wherever the `playground-elements` NPM package was installed (e.g. `./node_modules/playground-elements/playground-service-worker.js`) -- so that's what we want to set `sandboxBaseUrl` to. But it can be a little tricky to know exactly what that path would be. However, since we know the service worker and `playground-project.js` will be in the same place, we can use `import.meta.url` from `playground-project.js` to find that place. So changing the base URL to use `import.meta.url` instead of `document.location.href`, we can now pass `.` as the value of `sandboxBaseUrl` as a very easy way to say "use the service worker that's in the same directory you are".